### DISCUSSION TITLE
Make models orderable in the control panel

### DISCUSSION THREAD

**Discussion by janhenkes** [statamic-rad-pack/runway#631] [Feature Requests]
It would be nice to make models orderable in the control panel just like you can do with a collection.

**Comment by sandcatit-owner** [comment]
+1 for ability to add sorting in list view
Also would help #171

**Comment by duncanmcclean** [comment]
Added in [v9.7.0](https://github.com/statamic-rad-pack/runway/releases/tag/v9.7.0)
